packages/backend/src/email/client-stub.ts

```diff
@@ -42,6 +44,28 @@ export function createEmailClientStub(): EmailClientStub {
         },
       )
     },
+    async sendApprovalEmail(invitation: Invitation, entityId: string) {
+      const entityUrl = getEntityUrl(entityId)
+      await this.sendMailFromTemplate(
+        invitation.emailAddress,
+        'Congratulations! Your submission has been approved!',
+        './src/email/templates/approved.html',
+        {
+          entityUrl,
+        },
+      )
+    },
+    async sendRejectionEmail(invitation: Invitation) {
+      const { submitUiUrl } = getSubmitUrls(invitation)
+      await this.sendMailFromTemplate(
+        invitation.emailAddress,
+        'Sorry. Your submission has been rejected.',
+        './src/email/templates/rejected.html',
+        {
+          submitUiUrl,
+        },
+      )
+    },
```

Comment:
I already discuss this with @andrii-uhryn-absa He was supposed to communicate it with you, but you were on vacation, and the next day, yesterday, he was on vacation :)

We basically duplicate business logic here, that doesn't make any sense. We have to figure out a better way.

Maybe, creating some kind of `EmailService` for these functions, or keep them inside our higher-level modules like submission service and/or invitation service.

Reply:
Sure, I think it makes sense to be in the relevant service, i.e. submission service in this case.
But then that service will have a dependency on email client?

Reply:
Moved to `emailService`